### PR TITLE
Added feature to rust-fsm-dsl which adds #[repr(C)] to generated enum…

### PR DESCRIPTION
**Pull request by jost-aaron**
…s and structs so they can be used in FFI applications.

### CODE REVIEW

**Review by eugene-babichenko — COMMENTED** [review]
I would rather have this configurable like this is done with derives, e.g.

```rust
state_machine! {
    repr_c(true)
    ...
}
```

**Comment by jost-aaron** [comment]
Ok I changed my implementation to match your suggestion.

File: rust-fsm-dsl/Cargo.toml [hunk 1]
```diff
@@ -8,7 +8,7 @@ readme = "../README.md"
 license = "MIT"
 categories = ["data-structures", "rust-patterns"]
 keywords = ["fsm"]
-version = "0.6.1"
+version = "0.6.0"
```
Comment (jost-aaron) [inline rust-fsm-dsl/Cargo.toml:11]: I removed my version number increment. Thats probably something the maintainer should do.

File: rust-fsm/Cargo.toml [hunk 2]
```diff
@@ -18,7 +18,7 @@ std = []
 dsl = ["rust-fsm-dsl"]
 
 [dependencies]
-rust-fsm-dsl = { path = "../rust-fsm-dsl", version = "0.6.1", optional = true }
+rust-fsm-dsl = { path = "../rust-fsm-dsl", version = "0.6.0", optional = true }
```
Comment (jost-aaron) [inline rust-fsm/Cargo.toml:21]: Removed version increment as that should be done by the maintainer

**Pull request merged**